src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -565,13 +565,21 @@ def normalize_connection_config(connection):
             "organization": connection.organization,
             "base_url": connection.base_url
         }
+    elif isinstance(connection, ServerlessConnection):
+        # append "/v1" to ServerlessConnection api_base so that it can directly use the OpenAI SDK.
+        base_url = connection.api_base + "/v1"
```

Comment:
I think existing users already add a /v1 suffix in their api_base, do we need to consider the backward-compatibility?